Neos.Fusion/Classes/Core/Cache/ParserCache.php

```diff
@@ -105,4 +108,29 @@ private function getAbsolutePathForPackageRessourceUri(string $requestedPath): s
         $package = $this->packageManager->getPackage($resourceUriParts['host']);
         return Files::concatenatePaths([$package->getResourcesPath(), $resourceUriParts['path']]);
     }
+
+    /**
+     * Uses the same technique to resolve a package resource URI like Flow.
+     *
+     * package://My.Site/NodeTypes/Foo/Bar.fusion
```

Comment:
Will this become:
```suggestion
     * nodetypes://My.Site/Foo/Bar.fusion
```
?